src/search/FindInFilesUI.js

```diff
@@ -228,7 +230,7 @@ define(function (require, exports, module) {
                 promise: candidateFilesPromise
             };
 
-            filterPicker = FileFilters.createFilterPicker(exclusionsContext);
+            filterPicker = FileFilters.createFilterPicker(exclusionsContext, _findBar);
```

Comment:
FileFilters can not dispatch events now. You could make an event dispatcher by adding something like
EventDispatcher.makeEventDispatcher(exports) in FileFilters;